lib/KTable/KTableGridItem.vue

```diff
@@ -85,4 +93,8 @@
     white-space: normal;
   }
 
+  td {
+    border-bottom: 1px solid;
```

Comment:
The separation lines have been applied correctly. It looks good, thank you! 

Is there a particular reason why the border-bottom style is applied both here and on line 65? Just wondering if we can remove this second application since it gets overwritten.